app/assets/javascripts/templates.js

```diff
@@ -0,0 +1,7 @@
+window.templates = {};
```

Comment:
Compiled templates shouldn't be committed. Can you rebase and remove?

Reply:
removed